### DISCUSSION TITLE
Resetting TabView or Accordion in Dialog

### DISCUSSION THREAD

**Discussion by johnmanko** [primefaces/community#372] [PrimeNG]
I've got a TabView and Accordion inside a modal Dialog.  When interacting with either control, and close/reopen the dialog, the state of TabView and Accordian are not reset.  Specifically, the TabView doesn't reset to default on the first tab, and the Accordion shows with the first and last-view panels open.  Neither `ngOnInit` or `ngAfterViewInit` are called.

How can I reset both of those to a clean state?

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue clearly?

**Reply by johnmanko** [reply]
I'll get back to you on that.

**Comment by cetincakiroglu** [comment]
Closing the topic due to the inactivity.
